libs/backend/src/ballot_package/ballot_package_io.ts

```diff
@@ -51,7 +51,9 @@ export async function readBallotPackageFromUsb(
 
   const files = await fs.readdir(directoryPath, { withFileTypes: true });
   const ballotPackageFiles = files.filter(
-    (file) => file.isFile() && file.name.endsWith('.zip')
+    // OsX hidden files prefixed with `._` are not valid zip files and will break zip parsing
+    (file) =>
+      file.isFile() && !file.name.startsWith('._') && file.name.endsWith('.zip')
```

Comment:
I think it'd be reasonable to exclude anything starting with `.` since they're generally considered "hidden".